bedevere/util.py

```diff
@@ -3,7 +3,16 @@
 
 import gidgethub
 
+
+DEFAULT_BODY = ""
+TAG_NAME = "issue-number"
 NEWS_NEXT_DIR = "Misc/NEWS.d/next/"
+CLOSING_TAG = f"<!-- /{TAG_NAME} -->"
+BODY = f"""\
+{{body}}
+gh-{{issue_number}}
+{CLOSING_TAG}
```

Comment:
Why is there a closing tag but no opening tag?

I'm not sure what the tag is for, but my best guess is that it's so the bot can reliably recognize that it has already added a link. So maybe we should also add an open tag and use that to check whether to add the issue number.

Reply:
Taken from [2aee86a31df388b9292a28afb1c28a473c4d577b] (https://github.com/python/bedevere/blob/2aee86a31df388b9292a28afb1c28a473c4d577b/bedevere/bpo.py) there was no opening tag their either.

Reply:
@JelleZijlstra, can this conversation be resolved?

Reply:
Though it seems we're not using the tag now. If so, we should remove the closing tag too.